go.mod

```diff
@@ -1,39 +1,44 @@
 module github.com/Digital-Insight-Technologies-Ltd/gin-jwt-cognito
 
-go 1.20
+go 1.21.0
+
+toolchain go1.23.2
 
 require (
-	github.com/gin-gonic/gin v1.9.1
+	github.com/gin-gonic/gin v1.10.0
 	github.com/golang-jwt/jwt/v5 v5.2.1
+	github.com/google/uuid v1.6.0
```

Comment:
Pin dependency versions to patch releases or commit hashes to ensure build reproducibility and prevent unexpected breaking changes from future updates. [medium]